### PR TITLE
vuln(NSWG-ECO-420): html-pages

### PR DESCRIPTION
**Pull request by cjihrig**

### CODE REVIEW

File: vuln/npm/420.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "publish_date": "2018-05-09",
+  "author": "bl4de (https://twitter.com/_bl4de)",
+  "module_name": "html-pages",
+  "cves": ["CVE-2018-3744"],
+  "vulnerable_versions": "<=2.0.9",
+  "patched_versions": ">=2.1.0",
+  "recommendation": "Update to html-pages@2.1.0 or higher.",
+  "references": "- https://hackerone.com/reports/306607",
+  "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:H/I:N/A:N",
+  "cvss_score": 7.4,
```
Comment (lirantal) [inline vuln/npm/420.json]: should be 9.5

Comment (lirantal) [reply vuln/npm/420.json]: also, I updated the report in H1 to reflect the same vector scoring presented here

File: vuln/npm/420.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "coordinating_vendor": ""
```
Comment (ChALkeR) [inline vuln/npm/420.json]: Should be `null`, empty strings are not allowed.

**Comment by cjihrig** [comment]
Nits addressed.

File: vuln/npm/420.json (hunk 1 above)
Comment (ChALkeR) [reply vuln/npm/420.json]: How is this 9.5?

**Pull request merged**